stage2/include/climits

```diff
@@ -0,0 +1,6 @@
+#ifndef _CXXSHIM_CLIMITS
+#define _CXXSHIM_CLIMITS
+
+#include <limits.h>
+
```

Comment:
these appear to miss `std` aliases, might want to add them (similarly in other headers)

Reply:
climits and cfloat only have macros in them
cinttypes has std::imaxdiv_t which I added